addon/components/variable-plugin/autofilled/insert.hbs

```diff
@@ -0,0 +1,32 @@
+{{! @glint-nocheck: not typesafe yet }}
+<AuFormRow>
+  <VariablePlugin::Utils::LabelInput
+    @label={{this.label}}
+    @updateLabel={{this.updateLabel}}
+  />
+</AuFormRow>
+<AuFormRow>
+  <AuLabel for='autofill_key'>
+    {{t 'variable-plugin.autofill.autofillKey'}}
+  </AuLabel>
+  <AuNativeInput
+    id='autofill_key'
+    placeholder={{t 'variable-plugin.autofill.autofillKeyPlaceholder'}}
+    @type='text'
+    @width='block'
+    value={{this.autofillKey}}
+    {{on 'input' this.updateAutofillKey}}
+  />
+</AuFormRow>
+<AuFormRow>
+  <AuCheckbox
+    id='convert_to_string'
+    @checked={{this.convertToString}}
+    @onChange={{this.updateConvertToString}}
+  >
+    {{t 'variable-plugin.autofill.convertToString'}}
```

Comment:
When first looking at this, it wasn't clear to me what this meant. Maybe something that refers to leaving the variable node intact or not would be more clear? Maybe also an information text of some kind? As it is, it's not clear if, for example, this would be necessary to ensure that a date is inserted as a string rather than the timestamp.

Reply:
Changed the text, check if it's more clear or feel free to suggest one :)